File: ios/SwiftKotlination/SwiftKotlination/Sources/Features/Story/Model/Story.swift
struct Story: Codable, Equatable {
    let section: String
    let subsection: String
    let title: String
    let abstract: String
    let byline: String
    let url: String
    let multimedia: [Multimedia]
}

extension Story {
    func imageUrl(_ format: Multimedia.Format) -> String? {
        return multimedia.filter { $0.format == format }.first?.url
    }
}

File: android/SwiftKotlination/features/story/src/main/kotlin/fr/jhandguy/story/model/Story.kt
package fr.jhandguy.story.model

import kotlinx.serialization.Serializable

@Serializable
data class Story(
    val section: String = "",
    val subsection: String = "",
    val title: String = "",
    val abstract: String = "",
    val url: String = "",
    val byline: String = "",
    val multimedia: List<Multimedia> = emptyList()
)

fun Story.imageUrl(format: Multimedia.Format): String? = multimedia.firstOrNull { it.format == format }?.url
